app/models/space_contact.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+class SpaceContact < ApplicationRecord
+  belongs_to :space, optional: true
+  belongs_to :space_owner
```

Comment:
I think a contact does not BOTH belong to a :space and :space_owner, but to one of them! Or? My thinking is: Either it's contact information for 'Viken fylkeskommune' or it's contact information for 'Frederik II' - it's probably not for both? 

Seems we could use something called polymorphic associations to associate it with either of the two:
SO: https://stackoverflow.com/questions/6987336/rails-model-belongs-to-either-one-model-or-another
Rails guides: https://guides.rubyonrails.org/association_basics.html#polymorphic-associations

For example we could create a `:contactable_entity` that points to both :space and :space_owner  

Later, we want to be able to show contact information that is common to all spaces owned by space_owner. In the other cases we do this by looking for the same field on Space and SpaceOwner. (So on rendering Space#show, we both pull `Space.who_can_use` and `Space.space_owner.who_can_use` and display them seperately). 

For SpaceContact it would be nice to have the same interface, and I think the suggested changes above would allow that while still allowing a SpaceContact to only be connected to one entity and not both to a Space and SpaceOwner.

Reply:
Ok, makes sense to make it polymorphic.